CHANGELOG.md

```diff
@@ -0,0 +1,154 @@
+# Changelog
+
+## v3.3.0 (2025-10-01)
+
+### Tracer
+* [refactor][breaking] Have 'analyze_plots' return id, not full metadata
+* [refactor][breaking] Rename list_plots_analyses_report_types to list_plots_analysis_report_types
+* [refactor][breaking]  change order of arguments to allow future "url flattening"
```

Comment:
url flattening isn't future anymore and we could be specific, like:
- `change order and deprecate arguments for the following functions:` and then just list the functions